tracing-effectful/tracing-effectful.cabal

```diff
@@ -0,0 +1,114 @@
+cabal-version:      3.0
+-- The cabal-version field refers to the version of the .cabal specification,
+-- and can be different from the cabal-install (the tool) version and the
+-- Cabal (the library) version you are using. As such, the Cabal (the library)
+-- version used must be equal or greater than the version stated in this field.
+-- Starting from the specification version 2.2, the cabal-version field must be
+-- the first thing in the cabal file.
+
+-- Initial package description 'tracing-effectful' generated by
+-- 'cabal init'. For further documentation, see:
+--   http://haskell.org/cabal/users-guide/
+--
+-- The name of the package.
+name:               tracing-effectful
+
+-- The package version.
+-- See the Haskell package versioning policy (PVP) for standards
+-- guiding when and how versions should be incremented.
+-- https://pvp.haskell.org
+-- PVP summary:     +-+------- breaking API changes
+--                  | | +----- non-breaking API additions
+--                  | | | +--- code changes with no API change
+version:            0.1.0.0
+
+-- A short (one-line) description of the package.
+-- synopsis:
+
+-- A longer description of the package.
+-- description:
+
+-- The license under which the package is released.
+license:            BSD-3-Clause
+
+-- The file containing the license text.
+license-file:       LICENSE
+
+-- The package author(s).
+author:             Adrien Duclos
+
+-- An email address to which users can send suggestions, bug reports, and patches.
+maintainer:         adrien.duclos@scrive.com
+
+-- A copyright notice.
+-- copyright:
+category:           Development
+build-type:         Simple
+
+-- Extra doc files to be distributed with the package, such as a CHANGELOG or a README.
+extra-doc-files:    CHANGELOG.md
+
```

Comment:
Please add the `tested-with` field (as is in `tracing.cabal`) and regenerate CI with `haskell-ci` (`haskell-ci regenerate` in the repo root will do the job).

Reply:
Done.

Reply:
Well, that "Done" was a bit optimistic ! Took me quite a while to get the CI to build everything - oddly enough, some of the build plans seemed to be different between the CI and my own machine, though I switched to the same GHC and cabal version as the ones that failed on the CI. But this is now fixed.